dotnet/src/SemanticKernel.Abstractions/Contents/AudioStreamContent.cs

```diff
@@ -29,4 +29,20 @@ public AudioStreamContent(Stream stream, string? modelId = null, IReadOnlyDictio
     {
         this.Stream = stream;
     }
+
+    /// <summary>
+    /// Converts an AudioStreamContent to AudioContent by loading the stream data into memory.
+    /// </summary>
+    /// <returns></returns>
+    public AudioContent ToAudioContent()
+    {
+        if (this.Stream is MemoryStream stream)
+        {
+            return new AudioContent(stream.ToArray());
+        }
+
+        using var memoryStream = new MemoryStream();
+        this.Stream.CopyTo(memoryStream);
+        return new AudioContent(memoryStream.ToArray());
```

Comment:
Maybe it's not big deal but using BinaryReader will be more efficient.
https://code-maze.com/create-byte-array-from-stream-in-csharp/